backend/runner/runner.go

```diff
@@ -104,8 +105,21 @@ func Start(ctx context.Context, config Config) error {
 	}
 	svc.state.Store(ftlv1.RunnerState_RUNNER_IDLE)
 
-	go rpc.RetryStreamingClientStream(ctx, backoff.Backoff{}, controllerClient.RegisterRunner, svc.registrationLoop)
-	go rpc.RetryStreamingClientStream(ctx, backoff.Backoff{}, controllerClient.StreamDeploymentLogs, svc.streamLogsLoop)
+	go func() {
+		// In some environments we may want a delay before registering the runner
+		// We have seen istio race conditions that we think this will help
+		startDelay := os.Getenv("RUNNER_START_DELAY")
+		if startDelay != "" {
+			delay, err := strconv.Atoi(startDelay)
+			if err != nil {
+				logger.Errorf(err, "could not parse RUNNER_START_DELAY")
```

Comment:
Super minor nit: since we're just using this for testing, but we could prefix this with `FTL_`
`FTL_RUNNER_START_DELAY`